resources/public/api/conf/1.0/schemas/retrieveReliefInvestments/retrieve_relief_investments_response.json

```diff
@@ -0,0 +1,215 @@
+            "description": "The date of the investment.",
+            "type": "string",
+            "example": "2018-04-06"
+          },
+          "amountInvested": {
+            "description": "The total amount invested.",
+            "type": "number",
+            "example": "2000.99"
+          },
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    },
+    "seedEnterpriseInvestment": {
+      "description": "Financial details about SEED investment income.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "uniqueInvestmentRef": {
+            "description": "The Unique Investment Reference (UIR) or the name of the authorising tax office as shown on the certificate.",
+            "type": "string",
+            "example": "12345678"
+          },
+          "companyName": {
+            "description": "The name of the company holding the investment.",
+            "type": "string",
+            "example": "Goldman Sachs"
+          },
+          "dateOfInvestment": {
+            "description": "The date of the investment",
+            "type": "string",
+            "example": "2018-04-06"
+          },
+          "amountInvested": {
+            "description": "The total amount invested.",
+            "type": "number",
+            "example": "2000.99"
+          },
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the tax year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    },
+    "socialEnterpriseInvestment": {
+      "description": "Financial details about social enterprise investment income.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "uniqueInvestmentRef": {
+            "description": "The Unique Investment Reference (UIR) or the name of the authorising tax office as shown on the certificate.",
+            "type": "string",
+            "example": "12345678"
+          },
+          "socialEnterpriseName": {
+            "description": "The name of the investment or fund.",
+            "type": "string",
+            "example": "SE Inc"
+          },
+          "dateOfInvestment": {
+            "description": "The date of the investment.",
+            "type": "string",
+            "example": "2018-04-06"
+          },
+          "amountInvested": {
+            "description": "The total amount invested.",
+            "type": "number",
+            "example": "2000.99"
+          },
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the tax year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    }
+  },
+  "links": {
```

Comment:
links are in the wrong place, you also need to specify that the links are required and that the fields inside the links are required (see how it's done in another GET request schema)